### PR TITLE
Add update_investor/3 func

### PR DESCRIPTION
**Pull request by ktayah**

### CODE REVIEW

**Review by surrsurus — APPROVED** [review]

File: lib/belay_brokerage.ex [hunk 1]
```diff
@@ -35,6 +36,15 @@ defmodule BelayBrokerage do
     end
   end
 
+  @spec update_investor(String.t(), String.t(), investor()) :: {:ok, Investor.t()} | {:error, Ecto.Changeset.t()}
```
Comment (jeffdeville) [inline lib/belay_brokerage.ex:39]: Update and upsert? Seems confusing on when to use which

**Review by jeffdeville — CHANGES_REQUESTED** [review]
I think having an upsert and an update function is going to be confusing

File: lib/belay_brokerage.ex (hunk 1 above)
Comment (ktayah) [reply lib/belay_brokerage.ex:39]: I'm going to change the upsert to a create, we never needed the concept of an upsert. I just wanted to do it separately so I could update all our refs to point to the ref without adding breaking changes.
I already got the changes that replace the upsert with a create, I'll add it next

**Review by jeffdeville — APPROVED** [review]
Approve, pending the plan to resolve the upsert/update confusion

**Pull request merged**